apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -123,23 +157,23 @@ $cyan-idle: #0aa;
   .schedule-container {
     grid-area: schd;
     margin: 4vh 0;
-    font-family: digital-clock, monospace;
-    text-transform: uppercase;
 
     .onAir {
+      font-family: fourteen-seg;
       padding-bottom: 2vh;
       font-size: 15vh;
-      line-height: 0.9em;
+      line-height: 1.2em;
       color: var(--studio-active, $red-active);
 
       &--idle {
-        color: var(--studio-idle, $red-active);
+        color: $backlight;
+        // color: var(--studio-idle, $red-idle);
```

Comment:
Remember to remove unused code
We also need to maintain the variables where possible

Especially if we are merging to v2